cache/metercacher/metrics.go

```diff
@@ -78,7 +78,7 @@ func newMetrics(
 			Help:      "fraction of cache filled",
 		}),
 	}
-	return m, utils.Err(
+	return m, errors.Join(
```

Comment:
(style) for these metrics ones, we either do `errors.Join` inline in the return statement or assign a new `err` variable before the return statement. can we align them for consistency? I know this wasn't changed in this PR.

I think I would prefer assigning a new `err` variable before but don't have a strong opinion either way

Reply:
Going to leave this as-is because it's not something this PR changed. If we want we can do a follow-up to get this consistent.